### ISSUE TITLE
$ is not defined in leaflet.fusesearch.js:242 with L.CRS.Simple

### ISSUE THREAD

**Issue by 7Narwen** [naomap/leaflet-fusesearch#4]
Hello! I use leaflet 0.7 for game map with coordinates in pixels. It works, but fusesearch won't work with error "Uncaught ReferenceError: $ is not defined leaflet.fusesearch.js:242" (line "$(".result-item").remove();"). I’ve tried my geojson file in case of using OpenStreetMap for tileLayer and it works. Please, could you help?

```
var map = L.map('map', {
  crs: L.CRS.Simple,
  attributionControl: false,
  continuousWorld: true,
  maxZoom: 5, 
  minZoom: 5,
 }).setView([0, 0], 5);
 
var southWest = map.unproject([0, 7839], map.getMaxZoom());
var northEast = map.unproject([6572, 0], map.getMaxZoom());
map.setMaxBounds(new L.LatLngBounds(southWest, northEast));

var westeros = L.tileLayer('tiles/5/map_{x}_{y}.png', {continuousWorld: true});
westeros.addTo(map);

var searchCtrl = L.control.fuseSearch()
searchCtrl.addTo(map);

var featuresLayer = new L.GeoJSON(grrmmap.features, {
	pointToLayer: function (feature, latlng) {
		var newLatLng = map.unproject([latlng.lng, latlng.lat], map.getMaxZoom());
		return new L.marker(newLatLng);
	},
	onEachFeature: function (feature, layer, latlng) {
        feature.layer = layer;
		layer.bindPopup(feature.properties.description);
	}
});
searchCtrl.indexFeatures(grrmmap.features, ['name','description']);
map.addLayer(featuresLayer);
```

My geojson is:

```
var grrmmap = 
{
  "type": "FeatureCollection",
  "features": [
    {
      "type": "Feature",
      "geometry": {
        "type": "Point",
        "coordinates": [
          1108,
          1554
        ]
      },
      "properties": {
        "name": "Battle of the Fist",
		"english": "Battle of the Fist",
        "category": "war_wall",
        "description": "Battle of the Fist"
      }
    }, 
  ]
};
```

**Comment by naomap** [comment]
$ is a synonym for JQuery, which you need to import.  But this is a shame, this is only used in a couple of places and I should remove that dependency and use pure Javascript. I shall fix that in the summer, but not now as I'm off for a week. In the meantime just import jquery before you import the plugin.
Cheers, Antoine.

**Comment by 7Narwen** [comment]
Oh! Thank you very much!

**Comment by naomap** [comment]
Fixed by commit 4c6b43c7c4f72d53f7312c0ae5d034c26d93a787
